### PR TITLE
ISSUE #3444 - Add new styles to text fields, share boxes, search boxes, and select boxes

### LINKED ISSUE #3444: Updated style for text inputs/search bars
### Description

Adding the Groups feature into 3D Repo, allowing the user to create/edit/delete groups
Initially described in #3326. The text inputs, search bars, and select fields need to follow updated spec outline [here](https://www.figma.com/file/J5I3hYQTrLCjEVP1iqcyZK/3D-Viewer?node-id=36658%3A140).

### Goals
- [ ]  As a user I want o see a consistent clean ui

### Tasks
- [ ] Search bar outlined [here](https://www.figma.com/file/J5I3hYQTrLCjEVP1iqcyZK/3D-Viewer?node-id=36658%3A140)
	- on hover outline turns to base light
- [ ] Text inputs follow Search bar style outline [here](https://www.figma.com/file/J5I3hYQTrLCjEVP1iqcyZK/3D-Viewer?node-id=36658%3A140)
	- Labels should not change colour on hover/click/active

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #3444 

#### Description
- Update the styles of text fields, search boxes, and select boxes to match new style outlined [here](https://www.figma.com/file/J5I3hYQTrLCjEVP1iqcyZK/3D-Viewer?node-id=36658%3A140)

#### Test cases
- Try all different inputs (text field, share box, search box, select box)
- Try inputs in upload files modal
- Try inputs in the V4 viewer

### CODE REVIEW

**Review by george-hadfield — DISMISSED** [review]

File: frontend/src/v4/routes/components/criteriaField/criteriaField.styles.ts [hunk 1]
```diff
@@ -159,8 +159,7 @@ const emptySelectFieldValueStyleV4 = css`
 	opacity: 0.42;
 `;
 const emptySelectFieldValueStyleV5 = css`
-	/* TODO - fix after new palette is released */
-	color: #C1C8D5;
+	color: ${({ theme }) => theme.palette.base.lighter};
```
Comment (sanmont3drepo) [inline frontend/src/v4/routes/components/criteriaField/criteriaField.styles.ts:162]: Does this thing works in v4? theme.palette.base.lighter is from v5, right

File: frontend/src/v4/routes/components/textField/textField.styles.ts [hunk 2]
```diff
@@ -71,9 +71,7 @@ export const StyledMarkdownField = styled(MarkdownField)<{ $isPlaceholder?: bool
 		font-size: 14px;
 		overflow: hidden;
 
-		/* TODO: fix after new palette is released */
-		${({ $isPlaceholder }) => $isPlaceholder && css`color: #C1C8D5;`}
-
+		${({ $isPlaceholder }) => $isPlaceholder && css`color: ${({ theme }) => theme.palette.base.lighter};`}
```
Comment (sanmont3drepo) [inline frontend/src/v4/routes/components/textField/textField.styles.ts:74]: Same as previous comment, check if this works in v4

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

**Review by Amantini1997 — DISMISSED** [review]

File: frontend/src/v4/routes/components/criteriaField/criteriaField.styles.ts (hunk 1 above)
Comment (The-Daniel) [reply frontend/src/v4/routes/components/criteriaField/criteriaField.styles.ts:162]: I have tested and they both work

File: frontend/src/v4/routes/components/textField/textField.styles.ts (hunk 2 above)
Comment (The-Daniel) [reply frontend/src/v4/routes/components/textField/textField.styles.ts:74]: I have tested and they both work

**Comment by sanmont3drepo** [comment]
Textfields when they are not in edit mode dont look like textfields

image

**Comment by The-Daniel** [comment]
> Textfields when they are not in edit mode dont look like textfields
> 
> image

Fixed

**Review by sanmont3drepo — APPROVED** [review]

**Review by sanmont3drepo — APPROVED** [review]

**Comment by Amantini1997** [comment]
In any share modal, clicking on the icon, doesn't set the focus on the Textfield which doesn't get highlighted 
image

I'm not sure this is a problem, if not feel free to ignore this message

**Review by Amantini1997 — APPROVED** [review]

**Review by sanmont3drepo — APPROVED** [review]

**Review by george-hadfield — APPROVED** [review]

**Pull request merged**